config/config.go

```diff
@@ -56,39 +56,40 @@ type SourcesApiConfig struct {
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "LogLevelForSqlLogs", parsedConfig.LogLevelForSqlLogs)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "MarketplaceHost", parsedConfig.MarketplaceHost)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "AwsRegion", parsedConfig.AwsRegion)
-	//fmt.Fprintf(os.Stdout, "%s: %v\n", "AwsAccessKeyID", parsedConfig.AwsAccessKeyID)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "DatabaseHost", parsedConfig.DatabaseHost)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "DatabasePort", parsedConfig.DatabasePort)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "DatabaseName", parsedConfig.DatabaseName)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "FeatureFlagsEnvironment", parsedConfig.FeatureFlagsEnvironment)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "FeatureFlagsUrl", parsedConfig.FeatureFlagsUrl)
-	//fmt.Fprintf(os.Stdout, "%s: %v\n", "FeatureFlagsApiToken", parsedConfig.FeatureFlagsAPIToken)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "FeatureFlagsService", parsedConfig.FeatureFlagsService)
-	//fmt.Fprintf(os.Stdout, "%s: %v\n", "FeatureFlagsBearerToken", parsedConfig.FeatureFlagsBearerToken)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "CacheHost", parsedConfig.CacheHost)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "CachePort", parsedConfig.CachePort)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "SlowSQLThreshold", parsedConfig.SlowSQLThreshold)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "Psks", parsedConfig.Psks)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "BypassRbac", parsedConfig.BypassRbac)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "StatusListener", parsedConfig.StatusListener)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "BackgroundWorker", parsedConfig.BackgroundWorker)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "Migrations", parsedConfig.MigrationsSetup)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "MigrationsReset", parsedConfig.MigrationsReset)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "SecretStore", parsedConfig.SecretStore)
-	fmt.Fprintf(os.Stdout, "%s: %v\n", "TenantTranslatorUrl", parsedConfig.TenantTranslatorUrl)
+	fmt.Fprintf(&b, "%s: %s\n", "AppName", parsedConfig.AppName)
+	fmt.Fprintf(&b, "%s: %v\n", "Hostname", parsedConfig.Hostname)
+	fmt.Fprintf(&b, "%s: %v\n", "KafkaBrokers", parsedConfig.KafkaBrokers)
+	fmt.Fprintf(&b, "%s: %v\n", "KafkaTopics", parsedConfig.KafkaTopics)
+	fmt.Fprintf(&b, "%s: %v\n", "KafkaGroupID", parsedConfig.KafkaGroupID)
+	fmt.Fprintf(&b, "%s: %v\n", "MetricsPort", parsedConfig.MetricsPort)
+	fmt.Fprintf(&b, "%s: %v\n", "LogLevel", parsedConfig.LogLevel)
+	fmt.Fprintf(&b, "%s: %v\n", "LogLevelForMiddlewareLogs", parsedConfig.LogLevelForMiddlewareLogs)
+	fmt.Fprintf(&b, "%s: %v\n", "LogGroup", parsedConfig.LogGroup)
+	fmt.Fprintf(&b, "%s: %v\n", "LogHandler", parsedConfig.LogHandler)
+	fmt.Fprintf(&b, "%s: %v\n", "LogLevelForSqlLogs", parsedConfig.LogLevelForSqlLogs)
+	fmt.Fprintf(&b, "%s: %v\n", "MarketplaceHost", parsedConfig.MarketplaceHost)
+	fmt.Fprintf(&b, "%s: %v\n", "AwsRegion", parsedConfig.AwsRegion)
+	//fmt.Fprintf(&b, "%s: %v\n", "AwsAccessKeyID", parsedConfig.AwsAccessKeyID)
```

Comment:
it is not needed to add code which is commented out - only in rare situations if we will need to add such code later